src/dmd/dinterpret.d

```diff
@@ -826,13 +826,15 @@ private Expression interpretFunction(UnionExp* pue, FuncDeclaration fd, InterSta
             /* Value parameters
              */
             Type ta = fparam.type.toBasetype();
-            if (ta.ty == Tsarray && earg.op == TOK.address)
-            {
-                /* Static arrays are passed by a simple pointer.
-                 * Skip past this to get at the actual arg.
-                 */
-                earg = (cast(AddrExp)earg).e1;
-            }
+            if (ta.ty == Tsarray)
+                if (auto eaddr = earg.isAddrExp())
+                {
```

Comment:
Stylistically, an extra set of braces here so that the first if doesn't look out of place.

Reply:
It seems like clutter :-)